modules/apps/callbacks/go.mod

```diff
@@ -0,0 +1,170 @@
+module github.com/cosmos/ibc-go/modules/apps/callbacks
+
+go 1.20
+
+replace github.com/cosmos/ibc-go/v7 => ../../../
+
+replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
```

Comment:
slight nit: typically added after the required blocks

Reply:
ive seen that before -- but what is the reasoning for that?

i feel like it's actually nice to know up front which modules are getting replaced, rather than needing to scroll thru all the dependencies/transitivie dependencies.

Reply:
> ive seen that before -- but what is the reasoning for that?

`go mod edit` typically dumps them there, I guess adding them at the end makes sure you don't end up with a case of some `replace`s on top, others at the bottom.